server/models/Collective.js

```diff
@@ -653,8 +654,17 @@ export default function(Sequelize, DataTypes) {
             });
           }
 
+          if (instance.type === 'COLLECTIVE') {
```

Comment:
We should actually run that on all types of collectives (events, users, orgs...etc)